src/RTClib.h

```diff
@@ -413,6 +425,10 @@ class RTC_PCF8523 : RTC_I2C {
   void writeSqwPinMode(Pcf8523SqwPinMode mode);
   void enableSecondTimer(void);
   void disableSecondTimer(void);
+  void enableAlarmTimer(const DateTime &dt, const Pcf8523AlarmMode alarmMode,
+                        uint8_t alarmWeekday);
```

Comment:
This three-parameter version of `enableAlarmTimer()` has not been defined. It is not needed either.

Reply:
I think this was once against a result of issues I had with Git earlier that resulted in some changes getting lost. I was originally going to make the date setting explicit but felt it might be easier to just grab it from the DateTime like another alarm for one of the DS RTCs in the library was already doing. I'll review and fix. Thanks!!! :)